README.md

```diff
@@ -1,6 +1,17 @@
 # Pividori Lab Onboarding Repository
 
 The repository is home to the onboarding information for the [Pividori Lab](https://pivlab.org/) at the University of Colorado School of Medicine.
+
+# Checklist for new lab members
+* **Read the [`onboarding.md`](onboarding.md) document**.
+It has important information about the lab's mission, expectations, lab meetings, indiviudal meetings, source code, reproducibility, etc.
+Ask if you have questions.
+* Request access to the [lab's GitHub organization](https://github.com/pivlab).
+For this, Milton will need your GitHub user.
+* Request access to the [lab's Slack workspace](https://pivlab.slack.com/).
+It is usually a good idea to link this account to your CU email address.
+* 
+
```

Comment:
perhaps mention how to request access to the Alpine cluster or add the links with the information?
add how to set up CU VPN, anydesk VPN and rustdesk tunnel?

Reply:
Great point. I added a document at [docs/cu_dbmi.md](docs/cu_dbmi.md)